src/redux/home-page-slides/home-page-slides.sagas.js

```diff
@@ -0,0 +1,145 @@
+import { call, put, takeEvery } from 'redux-saga/effects';
+import { push } from 'connected-react-router';
+import {
+  removeSlideFromStore,
+  setAvailableSlides,
+  setSlide,
+  setSlideError,
+  setSlideLoading,
+  setSlides, setSlidesDrugAndDropList, setSlidesPagesCount
+} from './home-page-slides.actions';
+import {
+  createSlide, deleteSlide,
+  getAllAvailableSlides,
+  getAllSlides,
+  getSlideById,
+  updateSlide
+} from './home-page-slides.operations';
+import { setSnackBarMessage, setSnackBarSeverity, setSnackBarStatus } from '../snackbar/snackbar.actions';
+import {
+  ADD_SLIDE,
+  DELETE_SLIDE,
+  GET_AVAILABLE_SLIDES,
+  GET_SLIDE,
+  GET_SLIDES,
+  UPDATE_SLIDE, UPDATE_SLIDES_ORDER
+} from './home-page-slides.types';
+import { config } from '../../configs';
+
+const {
+  SUCCESS_ADD_STATUS,
+  SUCCESS_DELETE_STATUS,
+  SUCCESS_UPDATE_STATUS
+} = config.statuses;
+
+export function* handleSlidesLoad({ payload }) {
+  try {
+    yield put(setSlideLoading(true));
+    const slides = yield call(getAllSlides, payload.skip, payload.limit);
+    yield put(setSlidesPagesCount(Math.ceil(slides.count / payload.slidesPerPage)));
+    yield put(setSlides(slides.items));
+    yield put(setSlideLoading(false));
+  } catch (error) {
+    yield call(handleSlideError, error);
+  }
+}
+
+export function* handleAvailableSlides() {
+  try {
+    yield put(setSlideLoading(true));
+    const availableSlides = yield call(getAllAvailableSlides);
+    yield put(setAvailableSlides(availableSlides.items));
+    yield put(setSlidesDrugAndDropList(
+      [
+        { title: 'available', items: availableSlides.items.filter(el => el.show === true) },
```

Comment:
Why do you need this separation? I think - further it is easier to filter this list by a `show` field

Reply:
Fixed